rtt_rosclock/src/rtt_rosclock.cpp

```diff
@@ -55,13 +55,17 @@ const ros::Time rtt_rosclock::rtt_now()
   const uint64_t one_E9 = 1000000000ULL;
   // NOTE: getNSecs returns wall time, getTicks returns offset time
 #ifdef __XENO__
```

Comment:
That's very true, I'm closing this PR until I provide a new one with your recommendations :)